brave/brave-core
# Android syncv2 setup infobar

Resolves  https://github.com/brave/brave-browser/issues/10822

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues/10822) for my issue if one did not already exist.
- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
I. Do not show informer on clean install
1. Install this version 
2. Launch 
3. Ensure there is no infobar

II. Informer must be shown after update from sync v1 
1. Install old version with sync v1, like 1.11.96
2. Enable sync v1
3. Update with this PR version (some abobe 1.13.51)
4. Launch browser
5. Ensure informer is shown

III. Informer must not be shown when it was shown once
1. Follow steps (II.)
2. Close browser
3. Launch browser - informer must not be shown


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
